.github/workflows/maven_windows.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Maven build - Windows latest
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 14
+      uses: actions/setup-java@v1
+      with:
+        java-version: '14.0.1'
+    - name: Build with Maven
+      run: mvn -e -B package --file pom.xml
```

Comment:
do we need to package on all platforms?

Reply:
Interesting, I'm all up for changing it as it will reflect our current intent.
Just curious, will it save time as of now?

Reply:
Current maven build times:
Windows: 2m45s
Ubuntu: 1m:30s